server/app/graphql/resolvers/genes.rb

```diff
@@ -9,7 +9,7 @@ class Resolvers::Genes < GraphQL::Schema::Resolver
   scope { Gene.all }
 
   option(:ids, type: [String]) { |scope, value| scope.where(id: value)}
-  option(:name, type: String) { |scope, value| scope.where("name ILIKE ?", "#{value}%")}
+  option(:names, type: [String]) { |scope, value| scope.where(name: value) }
```

Comment:
@acoffman is this the correct activerecord call for matching against an array of names? (Also, does anyone have any theories why we were doing an SQL startswith query here?)